querymsc.py

```diff
@@ -505,5 +507,128 @@ def query_rain_rate_one_day_1_50(self, session):
                                   History.lon,
                                   History.station_id)
                  )
- 
-        return query
\ No newline at end of file
+  
+        return query
+
+class Gumbel:
+    """This class is meant to do all the fitting
+    of Gumbel distributions to extract design values.
+    
+    Attributes:
+        return_period (int): return period in years
+        min_fit (int): minimum number of years to consider
+            fitting Gumbel.
+    """
+
+    def __init__(self, return_period, min_fit=10):
+
+        self.return_period = return_period
+        self.min_fit = min_fit
+
+    def fit(self, x):
+        """Function to get L-moments to estimate the parameters
+        of a gumbel distribution. This method uses the 
+        right-skewed Gumbel distribution. Method mirrors 
+        Hosking 1990.
+        -----------------------------------
+        Args:
+            x (pandas Series): Series containing the annual
+                grouped extreme values for a given 
+                station over a range of years.
+            T (int): return period in years
+            N_min (int): Minimum number of years available to estimate
```

Comment:
Contains old parameter. Remove this because the `min_fit` class variable replaces this